docker/cuda/local_build.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
```

Comment:
why use this when there is a `Makefile`? The cuda libraries should be set to some sensible default in the `Makefile` and overridable via environment variables, exactly the same way as here.

Reply:
Sure - I only looked at the makefile afterwards; in fairness, the instructions say to use docker in the directory so that's where I went first..